### PR TITLE
Bump up content-atom version

### PR DESCRIPTION
**Pull request by marialivia16**
New fields have been added to the `content-atom` thrift model: https://github.com/guardian/content-atom/pull/39

### CODE REVIEW

**Comment by clloyd** [comment]
@paulmr @Reettaphant Could someone more familiar give this a quick review?

File: project/BuildVars.scala [hunk 1]
```diff
@@ -2,14 +2,14 @@ import sbt._
...
   lazy val scroogeVersion     = "4.2.0"
   lazy val akkaVersion        = "2.4.8"
   lazy val playVersion        = "2.5.3"
   lazy val mockitoVersion     = "2.0.97-beta"
 
   lazy val scanamoDeps = Seq(
-    "com.gu"                     %% "scanamo"              % "0.6.0",
-    "com.gu"                     %% "scanamo-scrooge"      % "0.1.2"
+    "com.gu"                     % "scanamo_2.11"          % "0.7.0",
+    "com.gu"                     % "scanamo-scrooge_2.11"  % "0.1.2"
```
Comment (paulmr) [inline project/BuildVars.scala]: can we put the `%%` back so that isn't specific to scala 2.11?

**Comment by paulmr** [comment]
Got it: the problem is actually that `scanamo-scrooge` is incompatible with `scanamo-0.7.0`.
I have created a fix: https://github.com/guardian/scanamo-scrooge/pull/5

**Comment by marialivia16** [comment]
@paulmr Were the tests working with your fix?

**Comment by paulmr** [comment]
@marialivia16 not quite: there are still a couple of issues. I am getting close though!

**Comment by paulmr** [comment]
to confirm: the issues are with `scrooge-scanamo`, not your PR. But your PR is a victim of those problems.

**Comment by marialivia16** [comment]
@paulmr Ok, cool. I'll wait for your changes then. Keep me updated. Thanks!

**Pull request merged**